lib/discourse_data_explorer/report_generator.rb

```diff
@@ -35,26 +35,10 @@ def self.generate_post(query_id, query_params, opts = {})
       build_report_post(query, table, attach_csv: opts[:attach_csv], result:)
     end
 
-    private
```

Comment:
Scoping keywords put in the class body don't affect class methods. All methods in this file are already public.